frontend-react/src/components/ManagePublicKey/ManagePublicKeyUpload.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useRef } from "react";
+import {
+    Button,
+    FileInput,
+    FileInputRef,
+    Form,
+    FormGroup,
+    Label,
+} from "@trussworks/react-uswds";
+
+export interface ManagePublicKeyFormProps {
+    onPublicKeySubmit: (e: React.FormEvent<HTMLFormElement>) => void;
+    onFileChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    onBack: () => void;
+    fileName: string;
+}
+
+const BASE_ACCEPT_VALUE = ".pem";
+
+export const ManagePublicKeyUpload = ({
+    onPublicKeySubmit,
+    onFileChange,
+    onBack,
+    fileName,
+}: ManagePublicKeyFormProps) => {
+    const isDisabled = fileName.length === 0;
+    const fileInputRef = useRef<FileInputRef>(null);
+
+    return (
+        <Form
+            name="managePublicKey"
+            onSubmit={(e) => onPublicKeySubmit(e)}
+            className="rs-full-width-form"
+        >
+            <FormGroup className="margin-bottom-3">
+                <Label
+                    className="font-sans-xs"
+                    id="upload-pem-input-label"
+                    htmlFor="upload-pem-input"
+                >
+                    Upload public key
+                    <br />
```

Comment:
Let's not use `<br>`s because they're literally only used for line breaks, which we can get with block-level elements.  Since this is wrapped in a `<label>`, though, we need to use `<span>`s instead of, say, `<p>`s:

```tsx
<Label>
    <span className="display-block">Upload public key</span>
    <span className="display-block text-gray-50">...</span>
</Label>
```